modules/loadbalancer/manifests/balance.pp

```diff
@@ -38,12 +38,17 @@
 # [*maintenance_mode*]
 #   Puts balancers into maintenance mode.
 #
+# [*aws_egress_nat_ips*]
+#   NAT Gateway Egress IPs from AWS Environment
+#
+
 define loadbalancer::balance(
     $servers,
     $deny_crawlers = false,
     $error_on_http = false,
     $https_port = 443,
     $internal_only = false,
+    $aws_egress_nat_ips = false,
```

Comment:
was this not supposed to be a list? Might be a bit confusing if you are using a variable as both a list and a boolean.

Reply:
maybe you want to use this code in your template `@aws_egress_nat_ips.length > 0`

Reply:
I think I can make that equal to undef here and that will clear it up..